src/virtualship/expedition/area_of_interest.py

```diff
@@ -0,0 +1,30 @@
+"""AreaOfInterest class."""
```

Comment:
"Area" may be a bit vague term. Do we use the word `Area`, `Region` or even `SpaceTimeRegion` throughout? Which term makes most sense? @ammedd, which would you prefer?

Reply:
Yes, let's do `SpaceTimeRegion`